pkg/virt-api/webhooks/mutating-webhook/preset.go

```diff
@@ -33,7 +33,12 @@ import (
 	"kubevirt.io/kubevirt/pkg/log"
 )
 
-const exclusionMarking = "virtualmachineinstancepresets.admission.kubevirt.io/exclude"
+const (
+	exclusionMarking   = "virtualmachineinstancepresets.admission.kubevirt.io/exclude"
+	namespaceKubevirt  = "kubevirt"
```

Comment:
namespace shouldn't be hardcoded here. We can detect the namespace virt-api is in and use that.  there's other places in the code that do this. we should have a helper function like "GetNamespace" somewhere to do this.